lib/helpers.js

```diff
@@ -64,11 +64,17 @@ export const stripTags = str => {
 export const markdownToText = str => {
   return stripTags(formatMarkdownValue(str, "html"))
 }
+export const parseFieldASTsIntoArray = fieldASTs => {
+  return map(flatMap(fieldASTs, "selectionSet.selections"), "name.value")
+}
 export const queriedForFieldsOtherThanBlacklisted = (fieldASTs, blacklistedFields) => {
   if (!fieldASTs) return true
-  const queriedFields = map(flatMap(fieldASTs, "selectionSet.selections"), "name.value")
+  const queriedFields = parseFieldASTsIntoArray(fieldASTs)
   return difference(queriedFields, blacklistedFields).length > 0
 }
+export const queryContainsField = (fieldASTs, soughtField) => {
+  return parseFieldASTsIntoArray(fieldASTs).includes(soughtField)
```

Comment:
FYI this seems to not work in ES5 (`Array.includes`). Did you mean to use the imported `lodash` version? (so something like `includes(parseFieldASTsIntoArray(fieldASTs), soughtField)`).